src/components/InterestForm/index.tsx

```diff
@@ -86,45 +108,47 @@ export default function InterestForm({ caseData }: { caseData: CaseListing }) {
     <FormContainer>
       <H3>Submit Interest</H3>
       {submitted ? (
-        <P>We have received your submission!</P>
+        <P>Your submission has been received!</P>
```

Comment:
can we add some padding bottom here? it looks a little squished atm
![image](https://github.com/calblueprint/immigration-justice-project/assets/67077248/16e94e53-c0c7-4594-bd0b-ce65326131be)

Reply:
`200px` should do

Reply:
sg, i changed it to `margin-bottom: auto` if that's ok? 
maybe it'll look better when the profile match is also there?

Reply:
actually that's a good point, we'll see and adjust after adding in profile match 👍
but having a `200px` minimum just in case won't hurt either